### DISCUSSION TITLE
How can i customize existing color variable?

### DISCUSSION THREAD

**Discussion by Louisecheow** [twbs/bootstrap#40533] [Q&A]
I want to change my primary color is yellow instead of changing in class `.btn-primary`. The color is successfully changed but still using existing color  

```scss
// Import Bootstrap's functions and variables first
@import "../node_modules/bootstrap/scss/functions";
@import "../node_modules/bootstrap/scss/variables";
@import "../node_modules/bootstrap/scss/mixins";

// Override Bootstrap's default variables
$primary: #ffcc00;

// Update theme colors map
$theme-colors: map-merge($theme-colors, ("primary": $primary));

// Import Bootstrap's rest of the styles
@import "../node_modules/bootstrap/scss/bootstrap";
```

![image](https://github.com/twbs/bootstrap/assets/138413636/d97e299c-4c50-462b-af47-518f15183d64)
![image](https://github.com/twbs/bootstrap/assets/138413636/027b33a1-4b46-47b4-b2df-28f1ed22a5fa)

---

_Edited by maintainer for syntax highlighting_

**Comment by julien-deramond** [comment]
Hi @Louisecheow 

I can't reproduce this issue locally with a local environment and the same Sass file that overrides the `$primary` color. Only one `.btn-primary` class is generated in the final CSS file, **so your solution seems to work pretty well**.

So, if it doesn't work on your side (buttons stay blue), it could be that the process generates either two `.btn-primary` classes in the final CSS file, or that you maybe load the basic Bootstrap CSS file + your own generated file.

Would you have more elements to share in order to look for other explanations?
